docs/topics/ref_yaml-template-for-custom-questionnaire.adoc

```diff
@@ -0,0 +1,115 @@
+            autoAnswerFor:
+              - category: Runtime
+                tag: Quarkus
+          - order: 2
+            text: Spring Boot
+            risk: green
+            rationale: Spring Boot is versatile and widely used.
+            mitigation: Ensure container compatibility.
+            applyTags:
+              - category: Runtime
+                tag: Spring Boot
+            autoAnswerFor:
+              - category: Runtime
+                tag: Spring Boot
+          - order: 3
+            text: Legacy Monolithic Application
+            risk: red
+            rationale: Legacy monoliths are challenging for cloud adaptation.
+            mitigation: Consider refactoring into microservices.
+      - order: 2
+        text: Does your application use a microservices architecture?
+        explanation: Assess if the application is built using a microservices architecture.
+        answers:
+          - order: 1
+            text: Yes
+            risk: green
+            rationale: Microservices are well-suited for cloud environments.
+            mitigation: Continue monitoring service dependencies.
+          - order: 2
+            text: No
+            risk: yellow
+            rationale: Non-microservices architectures may face scalability issues.
+            mitigation: Assess the feasibility of transitioning to microservices.
+          - order: 3
+            text: Unknown
+            risk: unknown
+            rationale: Lack of clarity on architecture can lead to unplanned issues.
+            mitigation: Conduct an architectural review.
+
+      - order: 3
+        text: Is your application's data storage cloud-optimized?
+        explanation: Evaluate if the data storage solution is optimized for cloud usage.
+        includeFor:
+          - category: Language
+            tag: Java
+        answers:
+          - order: 1
+            text: Cloud-Native Storage Solution
+            risk: green
+            rationale: Cloud-native solutions offer scalability and resilience.
+            mitigation: Ensure regular backups and disaster recovery plans.
+          - order: 2
+            text: Traditional On-Premises Storage
+            risk: red
+            rationale: Traditional storage might not scale well in the cloud.
+            mitigation: Explore cloud-based storage solutions.
+          - order: 3
+            text: Hybrid Storage Approach
+            risk: yellow
+            rationale: Hybrid solutions may have integration complexities.
+            mitigation: Evaluate and optimize cloud integration points.
+thresholds:
+  red: 1
+  yellow: 30
+  unknown: 15
+riskMessages:
+  red: Requires deep changes in architecture or lifecycle
+  yellow: Cloud friendly but needs minor changes
+  green: Cloud Native
+  unknown: More information needed
+----
+====
+
+[role="_additional-resources"]
+.Additional resources
+* xref:custom-questionnaire-fields_user-interface-guide[The custom questionnaire fields]
```

Comment:
On the [Preview ](https://deploy-preview-858--windup-documentation.netlify.app/docs/web-console-guide/master/#custom-questionnaire-fields_user-interface-guide) , 'The custom questionnaire fields'  is in purple.

Could this header be in black and bolder in order to be consistent with other headers ?

Reply:
If you mean fixing the header in the Additional resources section, then no..We do not use an y markup for Additional resources not to make links look weird .